### PR TITLE
removed warnings, added .clang-format, reformatted incoherent formatting

### PR DESCRIPTION
**Pull request by sauttefk**

### CODE REVIEW

**Comment by danielinux** [comment]
Thanks for the fixes @sauttefk -

Only one comment: Can we please spell the variable `daemonise` as `daemonize`

File: tty_attach.c [hunk 1]
```diff
@@ -1,164 +1,161 @@
...
-	memset(&sun, 0, sizeof(struct sockaddr_un));
-	sun.sun_family = AF_UNIX;
-	strncpy(sun.sun_path, path, strlen(path));
-	if(connect(connect_fd, (struct sockaddr *) &sun, sizeof(sun)) < 0){
-		perror("Cannot connect to socket");
-		exit(-1);
-	}
-	return connect_fd;
+static void usage(char *app) {
+  fprintf(stderr, "%s, Ver %s.%s.%s\n", basename(app), MAJORV, MINORV, SVNVERSION);
```
Comment (danielinux) [inline tty_attach.c:32]: indent: four spaces iso 2

File: tty_bus.c [hunk 2]
```diff
@@ -1,262 +1,251 @@
...
+        tty_bus_path = strdup(optarg);
+        break;
+      default:
+        usage(argv[0]);  // implies exit
+    }
+  }
+  if (optind < argc)
+    usage(argv[0]);  // implies exit
+
+  if (daemonise)
```
Comment (danielinux) [inline tty_bus.c:193]: spelling: daemonize

File: tty_fake.c [hunk 3]
```diff
@@ -32,153 +29,154 @@ static char *tty_bus_path;
...
-	}		
+
+int main(int argc, char *argv[]) {
+  int fd;
+  struct pollfd pfd[2];
+  int pollret, r;
+  char buffer[BUFFER_SIZE];
+  char *pts;
+  int ptmx;
+  int daemonise = 0;
```
Comment (danielinux) [inline tty_fake.c:77]: spelling: daemonize

**Review by danielinux — CHANGES_REQUESTED** [review]
please change the spelling of the variable daemonize

:gb: -> :us:

**Comment by sauttefk** [comment]
@danielinux 👍

**Pull request merged**